src/main/java/com/uid2/shared/Utils.java

```diff
@@ -184,4 +183,22 @@ public static <T1, T2> int upperBound(List<T1> list, T2 value, BiPredicate<T2, T
         }
         return first;
     }
+
+    //can make this as an config in admin/core service config
+    public static final String SiteSpecificDataSubDirPath = "sites/";
+    public static String getSiteSpecificMetadataPathName(int siteId, String metadataPathName)
+    {
+        return SiteSpecificDataSubDirPath +siteId + metadataPathName;
+    }
+
+    public static String getMetadataPathName(boolean isPublicOperator, int siteId, String metadataPathName)
```

Comment:
I'm thinking if this is  better done with an enum in case we have more different kinds of data in the future. This also directly reference operator in the name, which is more of an implementation detail, in reality this is site specific data. Eg in admin which will be reusing this nothing needs to make an assumption that an operator will be loading this data.

I suggest something like `enum DataScope { Global, Site }` (maybe can have a better name). Afterwards you could convert your boolean to an appropriate value in Core.

Reply:
ok sure will change that

Reply:
Will do this in the new UID2-575 ticket branch later after UID2-598 is done